### PR TITLE
added duckduckgo

### PR DESCRIPTION
**Pull request by helish88**

### CODE REVIEW

**Comment by parafoxia** [comment]
We don't accept PRs without a corresponding issue. Please create one.

**Comment by IkBenOlie5** [comment]
https://lmddgtfy.net/

Use this

File: carberretta/bot/cogs/links.py [hunk 1]
```diff
@@ -66,6 +66,13 @@ async def command_google(self, ctx: commands.Context, *, query: str) -> None:
             return await ctx.send("Your query should be no longer than 500 characters.")
 
         await ctx.send(f"<https://letmegooglethat.com/?q={query.replace(' ', '+')}>")
+    
+    @commands.command(name="duckduckgo", aliases=["ddg"])
+    async def command_google(self, ctx: commands.Context, *, query: str) -> None:
```
Comment (parafoxia) [inline carberretta/bot/cogs/links.py:71]:
```suggestion
    async def command_duckduckgo(self, ctx: commands.Context, *, query: str) -> None:
```

File: carberretta/bot/cogs/links.py [hunk 2]
```diff
@@ -66,6 +66,13 @@ async def command_google(self, ctx: commands.Context, *, query: str) -> None:
...
+        if len(query) > 500:
+            return await ctx.send("Your query should be no longer than 500 characters.")
+
+        await ctx.send(f"<https://duckduckgo.com/?q={query.replace(' ', '+')}>")
```
Comment (parafoxia) [inline carberretta/bot/cogs/links.py:75]: It would be better to use https://lmddgtfy.net/, as it then fits in more with the `+google` command.

**Review by parafoxia — CHANGES_REQUESTED** [review]

File: carberretta/bot/cogs/links.py (hunk 2 above)
Comment (helish88) [reply carberretta/bot/cogs/links.py:75]: agree

**Pull request closed without merging**